forge-gui/res/cardsfolder/upcoming/olorins_searing_light.txt

```diff
@@ -0,0 +1,11 @@
+Name:Olorin's Searing Light
+ManaCost:2 R W
+Types:Instant
+A:SP$ RepeatEach | RepeatPlayers$ Player.Opponent | RepeatSubAbility$ DBChooseCard | DamageMap$ True | SubAbility$ DBExile | SpellDescription$ Each opponent exiles a creature with the greatest power among creatures that player controls.\nSpell mastery — If there are two or more instant and/or sorcery cards in your graveyard, CARDNAME deals damage to each opponent equal to the power of the creature they exiled.
```

Comment:
why would this have a SubAbility?

Reply:
hmn actually, ChangeZoneEffect doesn't not support filling another ChangeZoneTable
but the exiling should be simultaneous
so maybe it has to look like this:
1. Repeat with ChooseCards
2. Exile
3. DealDamage

Reply:
That was my first version but I coudn't figure out making the damageamount  work on the dealdamage (since you somehow have to remember multiple variables from the repeat)

Reply:
hmn right, maybe it does need a second RepeatEach after all 🤔

Reply:
Is guess `SVar:X:RememberedLKI$FilterControlledByRemembered_CardPower
` should work ?

Reply:
in theory but you need to test

Reply:
Still having some problems with the damage part. I think imprinting the changed cards might work ?

Reply:
Seems like I got it working. 

Thanks for the help !

![image](https://github.com/Card-Forge/forge/assets/67333662/204bb76d-9da9-4c79-b4de-28c723d2ba16)